RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -292,31 +292,82 @@ boolean LX_getStarKey()
 	return autoAdv(1, $location[The Hole In The Sky]);
 }
 
-boolean beehiveConsider()
+boolean beehiveConsider(boolean at_tower)
 {
-	if(in_hardcore())
+	int damage_sources = 1; // basic hit
+	
+	// Familiars
+	if (have_familiar($familiar[shorter-order cook]) && auto_is_valid($familiar[shorter-order cook]))
 	{
-		if(have_skill($skill[Shell Up]) && have_skill($skill[Sauceshell]))
-		{
-			set_property("auto_getBeehive", false);
-		}
-		else
-		{
-			set_property("auto_getBeehive", true);
-		}
+		damage_sources += 5;
 	}
-	else
+	else if (have_familiar($familiar[mu]) && auto_is_valid($familiar[mu]))
+	{
+		damage_sources += 5;
+	}
+	else if (have_familiar($familiar[imitation crab]) && auto_is_valid($familiar[imitation crab]))
+	{
+		damage_sources += 4;
+	}
+	
+	// Combat skill to use
+	if (have_skill($skill[kneebutt]) || have_skill($skill[headbutt]))
+	{
+		damage_sources += 1;
+	}
+	
+	// Retatiatory skills
+	foreach sk in $skills[the psalm of pointiness, spiky shell, scarysauce, Jalape&ntilde;o Saucesphere]
 	{
-		if(have_skill($skill[Shell Up]) || have_skill($skill[Sauceshell]))
+		if (have_skill(sk))
 		{
-			set_property("auto_getBeehive", false);
+			damage_sources += 1;
 		}
-		else
+	}
+	
+	// Damage skills
+	foreach sk in $skills[dirge of dreadfulness, icy glare]
+	{
+		damage_sources += 1;
+	}
+	// Sleaze and stench will be taken care of war gear.
+	damage_sources += 2;
```

Comment:
only one, you can't wear both war gear

Reply:
You wear one outfit and one enemy drop.

For example, if you fight for frat, you wear the frat outfit for sleaze, and then use lockstock sandals or gaia beads for the stench damage.